zkvm/src/ops.rs

```diff
@@ -6,6 +6,8 @@ use crate::encoding::SliceReader;
 use crate::errors::VMError;
 use crate::types::Data;
 
+pub struct Program(Vec<Instruction>);
```

Comment:
in the spirit of upcoming https://github.com/interstellar/slingshot/issues/114 let's add docs to the things.

```suggestion
/// A builder type for assembling a sequence of `Instruction`s with chained method calls.
/// E.g. `let prog = Program::new().push(...).input().push(...).output(1).to_vec()`.
pub struct Program(Vec<Instruction>);
```